WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/EUCustomsScenarioValidator.swift

```diff
@@ -7,37 +7,47 @@ import Yosemite
     }
 
     private static let usCountryISOCode: Set<String> = ["US", "USA"] // United States of America
-    private static let countriesFollowingEUCustoms: Set<String> = ["AT", "AUT", // Austria
-                                                                   "BE", "BEL", // Belgium
-                                                                   "BG", "BGR", // Bulgaria
-                                                                   "HR", "HRV", // Croatia
-                                                                   "CY", "CYP", // Cyprus
-                                                                   "CZ", "CZE", // Czech Republic
-                                                                   "DK", "DNK", // Denmark
-                                                                   "EE", "EST", // Estonia
-                                                                   "FI", "FIN", // Finland
-                                                                   "FR", "FRA", // France
-                                                                   "DE", "DEU", // Germany
-                                                                   "GR", "GRC", // Greece
-                                                                   "HU", "HUN", // Hungary
-                                                                   "IE", "IRL", // Ireland
-                                                                   "IT", "ITA", // Italy
-                                                                   "LV", "LVA", // Latvia
-                                                                   "LT", "LTU", // Lithuania
-                                                                   "LU", "LUX", // Luxembourg
-                                                                   "MT", "MLT", // Malta
-                                                                   "NL", "NLD", // Netherlands
-                                                                   "NO", "NOR", // Norway
-                                                                   "PL", "POL", // Poland
-                                                                   "PT", "PRT", // Portugal
-                                                                   "RO", "ROU", // Romania
-                                                                   "SK", "SVK", // Slovakia
-                                                                   "SI", "SVN", // Slovenia
-                                                                   "ES", "ESP", // Spain
-                                                                   "SE", "SWE", // Sweden
-                                                                   "CH", "CHE"] // Switzerland
+}
+
+extension Country {
+    /// EU Country Code definitions.
+    ///
+    static let EUCountryCodes: Set<String> = ["AT", "AUT", // Austria
```

Comment:
Sorry for chiming in, but not all the countries on this list are EU: Norway and Switzerland are not.

Although it doesn't change anything in the scope of the Privacy Choices project, the name of this list might bring some confusion in the future, in different projects.

Reply:
Ooh, good catch! That list is the countries that [follow the EU customs rules](https://www.usps.com/international/new-eu-customs-rules.htm#countries), so the previous name (`countriesFollowingEUCustoms`) is more accurate. We could split up that list so we have 3 lists: EU countries only, countries following EU customs, and GDPR countries. (Although in that case we'll have to be careful to keep the EU countries list up to date 😅 )

Reply:
Nice catch, I have reverted the name to countriesFollowingEUCustoms.

Reply:
Thanks!

Now, when I think of it, should we show the banner for countries outside the EU (except GB, as you explained)? 🤔

Reply:
WPCom shows the banner for GDPR countries, but I just found out that they also add some extra ones.
I think we can copy [this list](https://github.com/Automattic/woocommerce.com/blob/1ccfd386088f594b85c00132fb535d13db08d3a4/plugins/wccom-plugins/cookie-terms/includes/class-wccom-cookie-terms.php#L68-L103).